application/src/main/java/org/togetherjava/tjbot/commands/tag/TagCommand.java

```diff
@@ -0,0 +1,88 @@
+package org.togetherjava.tjbot.commands.tag;
+
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.entities.Emoji;
+import net.dv8tion.jda.api.events.interaction.ButtonClickEvent;
+import net.dv8tion.jda.api.events.interaction.SelectionMenuEvent;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.CommandData;
+import net.dv8tion.jda.api.interactions.components.Button;
+import net.dv8tion.jda.api.interactions.components.ButtonStyle;
+import net.dv8tion.jda.api.interactions.components.selections.SelectOption;
+import net.dv8tion.jda.api.interactions.components.selections.SelectionMenu;
+import org.togetherjava.tjbot.commands.AbstractCommand;
+import org.togetherjava.tjbot.tag.TagSystem;
+
+import java.util.List;
+
+public class TagCommand extends AbstractCommand {
```

Comment:
Missing Javadoc, please explain the command, give some example discord usages and similar.

Reply:
added